examples/bpk-component-datepicker/examples.js

```diff
@@ -629,7 +629,6 @@ const DefaultVisualExample = () => (
       }}
       minDate={new Date(2020, 3, 1)}
       initiallyFocusedDate={new Date(2020, 3, 19)}
-      isOpen
```

Comment:
Removed this, so the calendar doesn't open by default and allows us to utilise the openEvents fix with a keyboard and improve A11y testing

Reply:
can we re-add this back? 
We plan to run an experiment where we test the focus behaviour :D

Reply:
This is just for the examples so doesn't affect production and is good for us to do for our examples